src/rendering/SceneManager.cc

```diff
@@ -538,6 +538,22 @@ rendering::MaterialPtr SceneManager::LoadMaterial(
       else
         ignerr << "Unable to find file [" << emissiveMap << "]\n";
     }
+
+    // light map
+    std::string lightMap = workflow->LightMap();
+    if (!lightMap.empty())
+    {
+      std::string fullPath = common::findFile(lightMap);
```

Comment:
Should we use `asFullPath` like the other maps are using?

Reply:
yep done. 1659774